cmd/createorupdate/createorupdate.go

```diff
@@ -92,6 +102,45 @@ func (s *server) validate(w http.ResponseWriter, r *http.Request) bool {
 	return true
 }
 
+func (s *server) handleDelete(w http.ResponseWriter, req *http.Request) {
+	authorizer, err := azureclient.NewAuthorizer(os.Getenv("AZURE_CLIENT_ID"), os.Getenv("AZURE_CLIENT_SECRET"), os.Getenv("AZURE_TENANT_ID"))
+	if err != nil {
+		resp := "500 Internal Error: Failed to determine request credentials"
+		s.log.Debug(resp, err)
+		http.Error(w, resp, http.StatusInternalServerError)
+		return
+	}
+	subID := os.Getenv("AZURE_SUBSCRIPTION_ID")
+	gc := resources.NewGroupsClient(subID)
```

Comment:
Why we dont move this to azureclient as all other? any particular reason?

Reply:
No particular reason, I am just not sure it's worth it since it's not going to be used by prod code and I don't know whether we want to spend time unit-testing the fake rp.

Reply:
true... dont think we will be mocking this one any time soon...